Second catch all PR for the tutorial

GiudGiud opened the pull request:
refs #21745

~~will be used to patch the material for day 2 & 3~~

GiudGiud committed:
Follow on the catch all PR for the tutorial
refs #21745

GiudGiud committed:
Use new sloshing pic

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 195da419f5a3b5a0890da69f3cbb04e34dccba59
+Subproject commit 3cd818f6f1ab70dd30c7d68c8aea6ba6f5251a6b
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1763156/) on 3cb3cb9 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/25443/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1763198/) on 3cb3cb9 wanted to post the following:


## Framework coverage

Inconsistent report tags were found between the head and base reports.
This can happen when reports are missing from either the head or the base.

Inconsistent tags:
  framework2
[Full coverage report](https://mooseframework.inl.gov/docs/PRs/25443/coverage/framework)


## Modules coverage

Inconsistent report tags were found between the head and base reports.
This can happen when reports are missing from either the head or the base.

Inconsistent tags:
  module-chemical_reactions
  module-combined
  module-contact
  module-contact-parallel
  module-electromagnetics
  module-external_petsc_solver
  module-fluid_properties
  module-fsi
  module-functional_expansion_tools
  module-geochemistry
  module-heat_conduction
  module-heat_conduction-parallel
  module-level_set
  module-misc
  module-navier_stokes
  module-navier_stokes-parallel
  module-navier_stokes-recover
  module-optimization
  module-optimization-parallel
  module-optimization-recover
  module-peridynamics
  module-peridynamics-parallel
  module-phase_field
  module-phase_field-parallel
  module-porous_flow
  module-porous_flow-parallel
  module-ray_tracing
  module-ray_tracing-parallel
  module-ray_tracing-recover
  module-rdg
  module-rdg-parallel
  module-reactor
  module-reactor-parallel
  module-richards
  module-scalar_transport
  module-scalar_transport-parallel
  module-scalar_transport-recover
  module-solid_properties
  module-solid_properties-parallel
  module-solid_properties-recover
  module-stochastic_tools
  module-stochastic_tools-parallel
  module-tensor_mechanics
  module-tensor_mechanics-parallel
  module-thermal_hydraulics
  module-thermal_hydraulics-parallel
  module-thermal_hydraulics-recover
  module-xfem
  module-xfem-parallel

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/25443/coverage/xfem)

</details>



This comment will be updated on new commits.

GiudGiud committed:
Adjust format of FEM slides a bit
Remove 'head in the sand' thing - sass has no place in the tutorial
Smooth transition to input file from numerical problem, refs #21745

GiudGiud committed:
Minor mods to equations

lindsayad commented on tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_multiphysics.md:
```diff
@@ -6,7 +6,7 @@
 - What about collaborating across research groups and/or disciplines?
 
-  - Head in the sand?
+  - Iterating between design teams?
```
This isn't clear to me. This sounds more like something that would happen at a tech company

GiudGiud replied:
Head in the sand was kind of sassy. People were finding their way around it, by iterating between the teams for TH, neutronics, fuel etc until everyone was happy enough

lindsayad commented on tutorials/darcy_thermo_mech/doc/content/workshop/problem/step01.md:
```diff
@@ -37,7 +37,7 @@ A basic MOOSE input file requires six parts, each of which will be covered in gr
 - `[BCs]`: Define the boundary condition(s) of the problem
 - `[Executioner]`: Define how the problem will be solved
-- `[Outputs]`: Define how the solution will be written
+- `[Outputs]`: Define how the solution will be returned
```
I honestly like `written` better. `returned` seems more in-code API like

GiudGiud replied:
We ll switch it back next time then
